docs/CodingStandards.md

```diff
@@ -140,7 +140,7 @@ Parameters should be commented in the main comment at
 the top of the function like this:
 
 ```
-// \param hatstand : [in/out] The Hatstand object to modify
+// @param hatstand : [in/out] The Hatstand object to modify
```

Comment:
Actually, the style should be:

`// @param [in/out] hatstand : The Hatstand object to modify`

Reply:
Ah, I had this fix but must have reverted it by accident. Good catch

Reply:
Done